### PR TITLE
[[ Bug 22711 ]] Fix crash when calling copy on non-field text chunk

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: docs/notes/bugfix-22711.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Fix crash when calling the `copy` command in an invalid container target
```
Comment (runrevmark) [inline docs/notes/bugfix-22711.md]:
```
# Fix crash when calling the `copy` command with a text chunk of a non-field object
```

**Review by runrevmark — CHANGES_REQUESTED** [review]
Please update the main commit title to match the (revised) PR title.

**Comment by runrevmark** [comment]
@livecode-vulcan review ok cfef977

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok cfef977e00a26d56f4a8c76247200b0be8fbac3c

**Comment by livecode-vulcan** [comment]
😎 test success cfef977e00a26d56f4a8c76247200b0be8fbac3c

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/840)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/249)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/249)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/249)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/249)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2366)
* try-community-universal-ios-iphoneos13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.5/builds/17)
* try-community-universal-ios-iphonesimulator13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.5/builds/17)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2394)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2382)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2381)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2416)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2356)

**Pull request merged**